### PR TITLE
Heketi-Templates:Persisting logfile in the heketidbstorage volume

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
Requires: #613 
Closes: #552 
Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

### CODE REVIEW

**Comment by MohamedAshiqrh** [comment]
@obnoxxx This has to be tested after #613 is merged.

File: extras/openshift/templates/heketi-template.json [hunk 1]
```diff
@@ -150,6 +150,10 @@
                   {
                     "name": "db",
                     "mountPath": "/var/lib/heketi"
+                  },
+                  {
+                    "name": "db",
```
Comment (lpabon) [inline extras/openshift/templates/heketi-template.json]: Same name?  I'm not sure that works.

Comment (obnoxxx) [inline extras/openshift/templates/heketi-template.json]: Right ... should it be s/th like "log" ?

**Comment by lpabon** [comment]
I think this change is not necessary if the log file is placed in `/var/lib/heketi` since that is the GlusterFS volume which is already mounted.  This is being fixed in #620

File: extras/openshift/templates/heketi-template.json (hunk 1 above)
Comment (MohamedAshiqrh) [inline extras/openshift/templates/heketi-template.json]: name should match name in ["volumes"](https://github.com/heketi/heketi/pull/619/files#diff-db4f124381d5c61236daeb291895e5f6R177). We can have a separate volumes "entry" for the same gluster(or heketidbstorage) volume, which internally will mount the same gluster(or heketidbstorage) volume on the host and the bind mount it for the different volume entry(which I feel unnecessary). I will test this out right away. I feel heketi container should be as light as possible.

**Comment by MohamedAshiqrh** [comment]
@lpabon I agree for now, but for long run it is better to have it in a place where it is expected. also for standalone heketi, it will be odd to have logs in `/var/lib/heketi/`.

**Comment by lpabon** [comment]
Just like #620, this change is *not* fixing the issue the correct way.   Please see #620 for more information.

**Comment by obnoxxx** [comment]
@MohamedAshiqrh is this something we should close?

**Comment by MohamedAshiqrh** [comment]
Not required any more as heketi is doing things as expected.

**Pull request closed without merging**